client/blocks/login/continue-as-user.scss

```diff
@@ -1,34 +1,68 @@
 .continue-as-user {
-	display: flex;
+	display: block;
 	animation: 1s appear 0s ease-in;
-	align-items: center;
-	justify-content: center;
-	flex-direction: column;
 	color: var( --color-text-subtle );
 	font-size: $font-body-small;
 	text-align: center;
+	height: calc( 70vh - 80px );
+	margin: 48px auto 0;
+	position: relative;
+	max-width: 400px;
+	min-height: 260px;
+
+	.continue-as-user__user-info {
+		height: auto;
+		align-self: flex-start;
+		margin: 0 auto;
+		display: flex;
+		flex-direction: column;
+
+		& > a.button {
+			margin: 0 20px;
+			border-radius: 4px; /* stylelint-disable scales/radii */
+		}
+	}
 
 	.continue-as-user__gravatar-link {
-		font-weight: 600;
 		color: var( --color-text );
+		margin: 0 auto;
+		line-height: 150%;
+
+		.continue-as-user__username {
+			margin-top: 12px;
+			font-size: $font-title-small;
+			font-weight: 600;
+		}
+		.continue-as-user__email {
+			margin-top: 4px;
+			font-size: $font-body;
+			margin-bottom: 48px;
+		}
 	}
 
 	.continue-as-user__gravatar {
+		display: inline;
 		border: 1px solid #a7aaad;
 	}
 
-	& > a.button {
-		margin: 20px 0;
-	}
-
 	.continue-as-user__change-user-link {
 		color: var( --color-link );
 		cursor: pointer;
+		margin: auto;
 
 		&:hover,
 		&:focus,
 		&:active {
 			color: var( --color-link-dark );
 		}
 	}
+
+	.continue-as-user__not-you {
+		display: block;
+		bottom: 0;
+		margin-bottom: -60px;
```

Comment:
This looks a bit fragile. Have we tested well on all possible screen sizes?

Reply:
Yes, I tested this on and adjusted the value to -40px now. So that we don't end up with a vertical scroll on smaller screens (iPhone 5)